tests/extproc/real_providers_test.go

```diff
@@ -50,6 +50,10 @@ func TestWithRealProviders(t *testing.T) {
 				CredentialFileLiteral: cc.AWSFileLiteral,
 				Region:                "us-east-1",
 			}}},
+			{Name: "aws-bedrock-anthropic", Schema: awsAnthropicSchema, Auth: &filterapi.BackendAuth{AWSAuth: &filterapi.AWSAuth{
```

Comment:
the change here is not used in any test case. Could you add one or two below in this file?

Reply:
or i think we can do that real provider test in a follow up (we don't have any existing one for anthropic message)